services/worker-manager/src/providers/aws.js

```diff
@@ -107,6 +89,25 @@ class AwsProvider extends Provider {
         ts.ResourceType === 'instance' ? instanceTags.concat(ts.Tags) : otherTagSpecs.push(ts)
       );
 
+      const userData = Buffer.from(JSON.stringify({
```

Comment:
Why was this moved?

Reply:
Because the workerconfig comes from the launchconfigs so it needs to be inside the loop